client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -68,10 +68,9 @@ class SiteTools extends Component {
 		const manageConnectionLink = `/settings/manage-connection/${ siteSlug }`;
 
 		const changeSiteAddress = translate( 'Change your site address' );
-		const startOver = translate( 'Delete your content' );
+		const startOver = translate( 'Reset your site' );
```

Comment:
@mpkelly @vykes-mac shouldn't this change be covered by the feature flag conditional instead of changed completely?